elixir/apps/api/lib/api/client/channel.ex

```diff
@@ -73,16 +78,37 @@ defmodule API.Client.Channel do
     end
   end
 
+  def handle_info(:config_changed, socket) do
+    :ok =
+      push(socket, "config_changed", %{
```

Comment:
@conectado one last change we need from connlib is to support `config_changed` (can be ignored for now) which will push a new interface config eg. whenever DNS settings are changed.

Reply:
+1 for not re-sending `init`.